src/lighteval/tasks/lighteval_task.py

```diff
@@ -162,7 +167,7 @@ def __init__(  # noqa: C901
                 containing task-specific functions. Defaults to None.
         """
         self.name = name
-        self.VERSION = 0
+        self.version = cfg.version  # TODO: this is changed from self.VERSION to lowercase - is the change consistent?
```

Comment:
using `version` instead of `VERSION` should work fine

Reply:
Ok. I removed the TODO comment.